libs/langgraph/langgraph/pregel/algo.py

```diff
@@ -182,11 +187,10 @@ def apply_writes(
         for chan in task.triggers
         if chan not in RESERVED and chan in channels
     }:
-        if channels[chan].consume():
-            if get_next_version is not None:
-                checkpoint["channel_versions"][chan] = get_next_version(
-                    max_version, channels[chan]
-                )
+        if channels[chan].consume() and get_next_version is not None:
```

Comment:
is this just stylistic or does it actually improve perf?

Reply:
No this was just style